src/frontend/js/contexts/SessionContext/BaseSessionProvider.tsx

```diff
@@ -70,6 +70,8 @@ const BaseSessionProvider = ({ children }: PropsWithChildren<any>) => {
     () => ({
       user,
       isLoadingUser,
+      joanieUser: null,
+      isLoadingJoanieUser: false,
```

Comment:
Hmmm... I do not like the idea to add joanie related properties into this provided which is supposed to fully ignored joanie.

Reply:
You ask to move this into Session context, how can i do it without adding these properties to `SessionContext` interface ?